glide.yaml

```diff
@@ -67,6 +69,9 @@ import:
 - package: github.com/bitly/go-simplejson
   version: ^0.5.0
 - package: github.com/google/go-querystring
+- package: github.com/pilu/config
```

Comment:
technically not part of this PR

Reply:
Absolutely true.  That's from PR #539, but I can't build tenant without it.